packages/components/organisms/f-footer/src/components/_tests/Footer.test.js

```diff
@@ -90,4 +91,97 @@ describe('Footer', () => {
         // Assert
         expect(wrapper.find('[data-test-id="country-selector"]').exists()).toBe(true);
     });
+
+    const localesExcludingES = [
+        'en-GB',
+        'en-AU',
+        'en-IE',
+        'en-NZ',
+        'it-IT'
+    ];
+
+    localesExcludingES.forEach(locale => {
+        it(`should not render the Confianza URL and icon when locale is ${locale}`, () => {
+            // Arrange & Act
+            propsData = {
+                ...propsData,
+                locale
+            };
+
+            const wrapper = shallowMount(Footer, { propsData });
+
+            // Assert
+            expect(wrapper.find('[data-test-id="confianza-link"]').exists()).toBe(false);
+            expect(wrapper.find('[data-test-id="confianza-icon"]').exists()).toBe(false);
+        });
+    });
```

Comment:
Minor comment, I think typically we would use Jest's own `it.each`. IIRC it can make debugging a little easier when they fail?
```suggestion
    it.each([
        'en-GB',
        'en-AU',
        'en-IE',
        'en-NZ',
        'it-IT'
    ])(`should not render the Confianza URL and icon when locale is %s`, locale => {
        // Arrange & Act
        propsData = {
            ...propsData,
            locale
        };

        const wrapper = shallowMount(Footer, { propsData });

        // Assert
        expect(wrapper.find('[data-test-id="confianza-link"]').exists()).toBe(false);
        expect(wrapper.find('[data-test-id="confianza-icon"]').exists()).toBe(false);
    });
```

Reply:
great spot, I completely forgot about that! thanks